packages/function-memoize-one/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "just-memoize-one",
+  "version": "1.0.0",
+  "description": "A memoize implementation that just remember the latest arguments and result",
+  "main": "index.js",
```

Comment:
add `"types": "index.d.ts",` after this line

Reply:
done!
since we are supporting types, doesn't make more sense to write the new packages in TypeScript?

Reply:
I don't want folks who don't use TS to have to transpile. I think supplying TS definitions is a good compromise